src/github.com/getlantern/flashlight/proxied/proxied.go

```diff
@@ -387,6 +387,10 @@ func chained(rootCA string, persistent bool) (http.RoundTripper, error) {
 			ClientSessionCache: clientSessionCache,
 		},
 	}
+	if persistent {
```

Comment:
Oh cool OK makes sense @oxtoacart! That actually answers some of my other questions about other places that use DisableKeepAlives -- all good.